res/css/views/elements/_Field.scss

```diff
@@ -89,6 +89,10 @@ limitations under the License.
     margin: 7px 8px;
     padding: 2px;
     pointer-events: none; // Allow clicks to fall through to the input
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    max-width: calc(100% - 18px);
```

Comment:
would be nice to have a comment for what the 18px is supposed to be

Reply:
Thanks, comment added, and I also realized it was meant to be 20px anyway...! 😭